pkg/runctl/run.go

```diff
@@ -30,7 +34,12 @@ func (r *tektonRun) GetStartTime() *metav1.Time {
 	if condition.IsUnknown() && condition.Reason != tekton.TaskRunReasonRunning.String() {
 		return nil
 	}
-	return r.tektonTaskRun.Status.StartTime
+	for _, step := range r.tektonTaskRun.Status.Steps {
+		if step.ContainerName == jfrStepName && step.Running != nil {
+			return &step.Running.StartedAt
+		}
```

Comment:
The container might have terminated already, in which case the start timestamp is in `step.Terminated.StartedAt` (see [here][1] and [here][2]).

[1]: https://github.com/tektoncd/pipeline/blob/e38d112e752ebbe97e4e3eccb1c271bbe5ecbcf7/vendor/k8s.io/api/core/v1/types.go#L2572-L2585
[2]: https://github.com/tektoncd/pipeline/blob/e38d112e752ebbe97e4e3eccb1c271bbe5ecbcf7/vendor/k8s.io/api/core/v1/types.go#L2549